Nuget/Ploeh.AutoFixture.nuspec

```diff
@@ -6,6 +6,9 @@
         <title>AutoFixture</title>
         <authors>Mark Seemann</authors>
         <owners>Mark Seemann</owners>
+        <dependencies>
+            <dependency id="Albedo" version="[1,2)" />

```

Comment:
I think we should have consistent versions for shared dependencies. If AutoFixture.Idioms works with Albedo 1.0.0, I think it'd be more correct to downgrade its dependency as well. I haven't tested, but looking at the change between 1.0.0 and 1.0.1, I think you're correct that it ought to work as well.